HW_1/task_3.py

```diff
@@ -19,3 +19,16 @@
 0
 Точка на отрицательной части оси абсцисс
 """
+x = int(input('Введите координату x: '))
+y = int(input('Введите координату y: '))
+
+if x == 0 and y == 0:
+    print("Точка в начале координат")
+elif x > 0 and y > 0:
```

Comment:
Подобные условия можно свернуть относительно повторяющегося значения.
```suggestion
elif x > 0 < y:
```